impeller/renderer/backend/vulkan/gpu_tracer_vk.cc

```diff
@@ -102,10 +102,6 @@ void GPUTracerVK::RecordEndFrameTime() {
   const auto device_holder = strong_context->GetDeviceHolder();
   if (!buffer->SubmitCommands([&, last_query,
                                device_holder](CommandBuffer::Status status) {
-        auto strong_context = context_.lock();
```

Comment:
We dont need this here, this might be causing shutdown issues since this may keep the context alive longer than it should